addons/project/static/tests/tours/project_task_history.js

```diff
@@ -88,9 +88,9 @@ registry.category("web_tour.tours").add("project_task_history_tour", {
         run: "click",
     }, {
         content: "Verify that 4 revisions are displayed (default empty description after the creation of the task + 3 edits)",
-        trigger: ".modal .html-history-dialog .revision-list .btn",
+        trigger: ".modal .html-history-dialog .revision-list .btn:not(.p-3)",
         run: function () {
```

Comment:
We should avoid using standard Bootstrap classes. If someone adds a p-3 class in the web editor, it will cause this tour to fail.